pkgs/development/libraries/ffmpegthumbnailer/default.nix

```diff
@@ -39,6 +39,11 @@ stdenv.mkDerivation rec {
       --replace '$'{exec_prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@
   '';
 
+  postInstall = ''
+    substituteInPlace $out/share/thumbnailers/ffmpegthumbnailer.thumbnailer \
+      --replace-fail 'ffmpegthumbnailer' "$out/bin/ffmpegthumbnailer"
```

Comment:
We should probably replace `=ffmpegthumbnailer`, otherwise this will get silently broken if the project switches to absolute path.